pytest_fixtures/api_fixtures.py

```diff
@@ -144,15 +163,15 @@ def module_os(
             entities.OperatingSystem()
             .search(
                 query={
-                    'search': 'family="Redhat" AND major="{0}" AND minor="{1}")'.format(
+                    u'search': u'family="Redhat" AND major="{0}" AND minor="{1}")'.format(
```

Comment:
These `u` should not be necessary, did you intentionally include them or did a formatting tool add them?

Reply:
Those are not necessary and I just forgot to `f-string` them.